pkg/registry/core/pod/strategy_test.go

```diff
@@ -116,14 +116,22 @@ func TestMatchPod(t *testing.T) {
 		},
 		{
 			in: &api.Pod{
-				Status: api.PodStatus{PodIP: "1.2.3.4"},
+				Status: api.PodStatus{
+					PodIPs: []api.PodIP{
```

Comment:
Test cases for multiple ips, and possibly ipv6 addresses?

Reply:
added